CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
```

Comment:
Can you add something like

+* Cromwell has an entirely new internal architecture which scales much better, and will allow for cleaner future extension